blog.html

```diff
@@ -0,0 +1,182 @@
+            <li><a href="#">Art (24)</a></li>
+            <li><a href="#">Robot (24)</a></li>
+            <li><a href="#">Launchpad (24)</a></li>
+            <li><a href="#">Crypto (410)</a></li>
+          </ul>
+        </nav>
+
+        <!-- Tags Section -->
+        <section>
+          <h3>Tags</h3>
+          <ul>
+            <li><a href="#">NFT</a></li>
+            <li><a href="#">ICO</a></li>
+            <li><a href="#">Robot</a></li>
+            <li><a href="#">Crypto</a></li>
+            <li><a href="#">Launchpad</a></li>
+            <li><a href="#">Metaverse</a></li>
+          </ul>
+        </section>
+      </aside>
+      <article>
+        <img src="basilisk-image.jpg" alt="The Basilisk" />
+        <p>Photo caption.</p>
+        <h1>The Basilisks NFT</h1>
+        <span>Gaming</span>
+        <p>
+          <strong>Tony Nguyen</strong><time datetime="2022-06-20">20 Jun 2022</time>
+        </p>
+        <p>
+          The Basilisks that <a href="#">players</a> collect are represented
+          with NFTs on Ethereum's blockchain with real-world value. Players
+          use the <a href="#">Basilisks they</a> have gathered to battle
+          other players to win ether (ETH).
+        </p>
+        <p>
+          In Balthazar Dragons, players are immersed in a 3D open world to
+          explore and capture dragon-like beasts called Basilisks. Balthazar
+          Dragons is an upcoming fantasy role-playing game developed on the
+          Ethereum blockchain by a decentralized autonomous organization
+          (DAO) called the Balthazar DAO.
+        </p>
+        <ul>
+          <li>List name 1</li>
+          <li>List name 2</li>
+          <li>List name 3</li>
+          <li>List name 4</li>
+        </ul>
+        <blockquote>
+          <p>
+            “The first huge metaverse arena to arrange live sports, hang out and
+            even make bets and wagers”
+          </p>
+          ⭐ 8.5/10 Markout Corporation
+        </blockquote>
+
+        <figure>
+          <img src="basilisk-image-2.jpg" alt="Basilisk Image 2" />
+
+          <img src="basilisk-image-3.jpg" alt="Basilisk Image 3" />
+          <figcaption> Basilisk Images</figcaption>
+        </figure>
+
+        <h2>NFTs are digital assets that represent real-world objects</h2>
+        <p>
+          Setting up your avatar and personalizing your nickname on Binance is
+          quick and easy...
+        </p>
+        <h3>NFT Avatar Generators</h3>
+        <a href="#">
+          <p>
+            You can find a range of NFT marketplaces where you can buy existing
+            NFTs or list your own NFTs for sale.
+          </p>
+        </a>
+        <p>
+          Tags: <a href="#" target="_blank"><span>NFT</span></a>
+          <a href="#" target="_blank"><span>ICO</span></a>
+          <a href="#" target="_blank"><span>Crypto</span></a>
+        </p>
+
+        <div>
+          <span>Share: </span>

```

Comment:
The `<span> `element should only be used for inline text styling inside a text block, not as a standalone text element.